src/features/device-provisioning/register.ts

```diff
@@ -49,21 +49,24 @@ export const register: RequestHandler = async (req, res) => {
 		// but clone to make sure it isn't propagated elsewhere
 		req = _.clone(req);
 		req.apiKey = _.cloneDeep(req.apiKey);
-		if (req.apiKey != null && req.apiKey.permissions != null) {
+		if (req.apiKey?.permissions != null) {
 			req.apiKey.permissions.push('resin.device.read');
 			req.apiKey.permissions.push('resin.device.create-device-api-key');
 		}
 
 		const response = await sbvrUtils.db.transaction(async (tx) => {
 			const device = await api.resin.post({
 				resource: 'device',
-				passthrough: { req, tx },
+				passthrough: {
+					req,
+					tx,
+				},
 				body: {
 					belongs_to__user: userId,
-					belongs_to__application: applicationId,
 					device_type: deviceType,
 					supervisor_version: supervisorVersion,
 					uuid,
+					belongs_to__application: applicationId,
```

Comment:
Does this need the famous `// TODO-MULTI-APP: This relies on the hook creating the device_application` comment?